_posts/2016-02-17-2016-roadmap.markdown

```diff
@@ -0,0 +1,100 @@
+
+*Goals*:
+
+- Make the Polymer library compatible with latest web component specs and shipping implementations
+  - Update styling system to be compatible with custom properties
+  - Update shady DOM to be compatible with shadow dom v1
+  - Update custom elements treatment to be compatible with custom elements v1
+
+### Address user feedback on data system
+
+One of the common recurring pieces of feedback we hear about Polymer is confusion in the data binding system. We’re looking to help make it clearer how to interface with data binding, to get all the lightweight sugar without some of the cognitive overhead when it comes to large, interconnect data models. This may come in the form of creating new library features, evolving existing features, building additional layers on top, or improving clarity in the Polymer documentation - or most likely some combination of all of the above.
+
+### Tooling
+
+The Polymer team works on [a](https://github.com/PolymerLabs/polyserve) [lot](https://github.com/Polymer/vulcanize) [of](https://github.com/PolymerLabs/polygit) [tools](https://github.com/PolymerLabs/crisper) to improve the development workflow and enable fast deployed sites. Many of these [tools](https://www.polymer-project.org/1.0/tools/overview.html) can get lost or buried, and it can be complicated to figure out how they might work together.
+
+We want to keep all of the tools modular, but for simplicity and better discovery we’re exploring building a wrapped-up “Polymer CLI,” that includes the tools we build for Polymer development. Currently we’re focusing only on Polymer-specific tools - for things like package management we want to continue to rely on the most commonly-used tools in the broader ecosystem that fit the development model.
+
+*Goal*:
+
+- Prototype Ultimate Polymer CLI, Tools
+
+### Community and Ecosystem
+
+The Polymer community has grown incredibly quickly and there are a number of individuals making extremely meaningful contributions to both the library and Polymer elements. We want to do more to highlight community work and generally better serve the Polymer and web components ecosystem. This means even more transparency around the core Polymer team’s roadmap, more regular announcement and updates, and promoting element ownership beyond just the core Polymer team.
+
+*Goals*:
+
+- Accelerate Polymer + Community engagement
+  - Publish all elements to npm and bower.
+  - Regularly publish project and roadmap updates
+  - Work with community members to help drive elements forward.
+  - Highlight awesome elements being built
```

Comment:
can we add a bullet about tightening the feedback loop. We should evolve polymer as our community evolves. I feel like _they_ should help us define a roadmap rather than us setting it in stone.